phpstan.neon.dist

```diff
@@ -16,6 +16,7 @@ parameters:
 		- **/view*.php
 		- include/widget-calendar.php
 		- install/plugin-updater.php
+	treatPhpDocTypesAsCertain: false
```

Comment:
Looks like it is not needed anymore, it was because PHPStan assumed that `$taxonomy` and `$this->pre_term_name` were string and that the following check was useless, see https://github.com/polylang/polylang/actions/runs/9562954062/job/26360443848?pr=1490.